src/risk_atlas_nexus/library.py

```diff
@@ -539,11 +540,28 @@ def identify_risks_from_usecases(
             "Usecases must be a list of string.",
         )
 
+        # For the given taxonomy type, check if the user has provided 'cot_examples'. If not,
+        # retrieve the default cot examples from the master. If no examples exist in the master,
+        # set it as None.
+        RISK_IDENTIFICATION_COT = load_resource("risk_generation_cot.json")
+        processed_examples = (
```

Comment:
Hi @ingelise ,
This will always return an empty `processed_examples` in the case the taxonomy is None. We should have `ibm-risk altas` as the default taxonomy, otherwise inferencing will fail when there are too many risks (risks from all taxonomies combined)  due to the model context limit restrictions.